samples/v1/types_config.go

```diff
@@ -64,6 +64,14 @@ type ConfigSpec struct {
 	// content but the operator will not recreate(or update) anything
 	// listed here.
 	SkippedTemplates []string `json:"skippedTemplates,omitempty" protobuf:"bytes,6,opt,name=skippedTemplates"`
+
+	// skippedhelmcharts specifies names of helmcharts that should NOT be
+	// created/updated.  Admins can use this to allow them to delete content
+	// they don’t want.  They will still have to manually delete the
+	// content but the operator will not recreate(or update) anything
+	// listed here.
+	// +listType=set
+	SkippedHelmCharts []string `json:"skippedHelmCharts,omitempty" protobuf:"bytes,7,rep,name=skippedhelmCharts"`
```

Comment:
why is this `rep` and also why is it not `opt`?

Reply:
Initially since the helmchart were to be installed by choice, i kept it as rep, as my understanding was i sholdn't keep it as optional since we need to install it.. but then i missed to change it back to opt after the discussion.. i have changed it now.